Neos.FluidAdaptor/Classes/Core/Parser/Interceptor/ResourceInterceptor.php

```diff
@@ -91,7 +91,9 @@ public function setDefaultPackageKey($defaultPackageKey)
      */
     public function process(NodeInterface $node, $interceptorPosition, ParsingState $parsingState)
     {
-        /** @var $node TextNode */
+        if (!$node instanceof TextNode) {
+            return $node;
+        }
```

Comment:
Why is that non breaky? Would unterstand the if-instanceof around the next lines but not before

Reply:
The next line always assumes its a TextNode `$node->getText()` but this might fail, as this is a generic processor. Instead when we encounter a node which is not a TextNode we will just early return instead of causing an error.